unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneController.cs

```diff
@@ -228,14 +228,14 @@ public bool ProcessMessage(QueuedSceneMessage_Scene msgObject, out CustomYieldIn
                     case MessagingTypes.ENTITY_CREATE:
                         {
                             if (msgPayload is Protocol.CreateEntity payload)
-                                scene.CreateEntity(payload.entityId);
+                                scene.CreateEntity(payload.entityId.GetHashCode());
```

Comment:
Do you mean to add new messages for special cases?

Reply:
but how do we ensure that `0xffffffff` is not a result from a sepecfic string in `entityId.GetHashCode()`?

I mean, imagine that `GetHashCode` of the string `"E8F"` gives the result `4294967295`  0xffffffff representation in long is `4294967295` 

This function will return the same number `4294967295` for the string "0" and for the string "E8F". It is very unlikely to happen but it can happen.


I can add the numbers of the GetHashCode of those strings, but if the GetHashCode implementation of the string class in the project changes, this won't work

Reply:
Brians says that the first 10 ids are reserved ids so I think we can safely do 

`
        private const long CONST_SCENE_ROOT_ENTITY = 0;
        private const long CONST_AVATAR_ENTITY_REFERENCE = 1;
        private const long CONST_AVATAR_POSITION_REFERENCE = 2;
        private const long CONST_FIRST_PERSON_CAMERA_ENTITY_REFERENCE = 3;
        private const long CONST_THIRD_PERSON_CAMERA_ENTITY_REFERENCE = 4;`